content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -157,7 +157,31 @@ It is then possible to switch to editing this layer by selecting it from the lay
 
 ![Select Layer][6]
 
-Tip: Layers animate a model in the order that they’re created in the layers panel. Any animation values they set on a model will be overwritten by subsequent layers if they are animating the same bones.
+### Layer Blending
+
+By default, layers animate a model in the order that they’re created in the layers panel. Any animation values they set on a model's bones will be overwritten by subsequent layers. If instead you wish to blend the animation values of the layers together, you can set the `blend type` of your layers to `Additive` rather than the default `Override`:
+
+![Layer Blend][11]
+
+The blend weight value of each layer is used when blending multiple layers to determine how much each layer should contribute to the final animation. These blend weights can be adjusted at runtime in your game scripts to update the blends of your layers in real time:
+```javascript
+var AnimationBlending = pc.createScript('animationBlending');
+
+// initialize code called once per entity
+AnimationBlending.prototype.initialize = function() {
+    this.entity.anim.on('smile_start', function (event) {
+        this.entity.anim.findAnimationLayer('smile').weight = 1;
+    }), this);
+    this.entity.anim.on('smile_end', function (event) {
+        this.entity.anim.findAnimationLayer('smile').weight = 0;
+    }), this);
+};
+```
+
+If you [mask your layers][/en/user-manual/animation/anim-layer-masking], you can set the `blend type` of your layers to `Additive` to blend in an animation which only controls part of your model's bones. Updating the `blend weight` in real time as described above can allow you to create smooth blends between animations on different layers. This is particularly useful when animation characters that need to carry out different actions using their upper and lower body. For example, you could have a `shooting` animation that is blended in and out on a characters upper body, while freeing up the lower body for various locomotion animations such as `walking` and `running`.
```

Comment:
```suggestion
If you [mask your layers][anim-layer-masking], you can set the `blend type` of your layers to `Additive` to blend in an animation which only controls part of your model's bones. Updating the `blend weight` in real time as described above can allow you to create smooth blends between animations on different layers. This is particularly useful when animation characters that need to carry out different actions using their upper and lower body. For example, you could have a `shooting` animation that is blended in and out on a characters upper body, while freeing up the lower body for various locomotion animations such as `walking` and `running`.
```

Reply:
Fixed link to the manual page

Reply:
```suggestion
If you [mask your layers][/en/user-manual/animation/anim-layer-masking], you can set the `Blend Type` of your layers to `Additive` to blend in an animation which only controls part of your model's bones. Updating the `Blend Weight` in real time as described above can allow you to create smooth blends between animations on different layers. 

This is particularly useful when animation characters that need to carry out different actions using their upper and lower body. For example, you could have a `shooting` animation that is blended in and out on a characters upper body, while freeing up the lower body for various locomotion animations such as `walking` and `running`.
```

Reply:
Maybe GIFs of what the changing the blend weight could look like at the extremes (0, 0.5, 1.0)? There will be some developers who have never worked with layer masks and blend weights before so a visual example would help